themes/edx.org/lms/templates/course_modes/_upgrade_button.html

```diff
@@ -22,4 +22,15 @@
         (<span class="upgrade-price-string">$${min_price} USD</span>)
         % endif
         </button>
-</li>
\ No newline at end of file
+</li>
+
+<%static:require_module_async module_name="js/commerce/track_ecommerce_events" class_name="TrackECommerceEvents">
+var upgradeLink = $("#track_sel_upgrade");
+
+TrackECommerceEvents.trackUpsellClick(upgradeLink, 'track_selection', {
+    pageName: "track_selection",
+    linkType: "button",
+    linkCategory: "(none)"
```

Comment:
What does linkCategory do/how is it used? Would having none cause any issues?

Reply:
Sorry for the confusion- the category is a way to group similar links that appear on different pages, for example the EDXWELCOME banner that appears on a couple pages shares the same category. That worked out for *most* of the links, but there were a few that were one-of-a-kind, where we left the category as "(none)" for consistency. See more info here: 
https://docs.google.com/document/d/13Fwl3x-XUDGwm73ftkiKgiruiK3wWnlHeQ3mVSl4q_8/edit?ts=5f0cc156

Reply:
Could we remove the linkCategory key/value?